### DISCUSSION TITLE
How do I turn off the leaderboard in a multilobby when not spectating but playing?

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#22212] [Q&A] [answered]
I dont want to see the current place I have, and even if i do the shift tab combo and set it to never I still see this. what can I do?

**Comment by bdach** [comment, marked as answer]
I believe the leaderboard being always shown in multiplayer is intentional in order to indicate that it's multiplayer, and stable also does this.

I'm aware that there is a significant amount of people saying that shouldn't be the case, I'm just stating how the situation looks right now.
